static/app/views/issueDetails/toggleButton.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import {t} from 'sentry/locale';
+
+interface IssueDetailsToggleButtonProps {
+  isShown: boolean;
+  setShownState: (boolean) => void;
```

Comment:
this is pretty nit picky - but props that take a function usually follow the `onClick` style. So maybe `onShownChange` or something
```suggestion
  onShownChange: (boolean) => void;
```